test-utils/loadtester/src/main.rs

```diff
@@ -158,92 +84,32 @@ fn main() {
-    }
-}
-
-fn load_state_dump(matches: &clap::ArgMatches<'_>) {
-    let dir_buf = value_t_or_exit!(matches, "home", PathBuf);
-    let state_dump_path = value_t_or_exit!(matches, "state_dump", PathBuf);
-    let dir = dir_buf.as_path();
-    let state_dump = state_dump_path.as_path();
-    let store = create_store(&get_store_path(dir));
-    store.load_from_file(ColState, state_dump).expect("Failed to read state dump");
-}
-
 fn run(matches: &clap::ArgMatches<'_>) {
     let n = value_t_or_exit!(matches, "accounts", u64);
-    let prefix = value_t_or_exit!(matches, "prefix", String);
-    let massive_accounts = matches.is_present("massive_accounts");
     let tps = value_t_or_exit!(matches, "tps", u64);
     let duration = value_t_or_exit!(matches, "duration", u64);
     let transaction_type = value_t_or_exit!(matches, "type", TransactionType);
 
-    let addr: String;
-    let addrs: Vec<_>;
-    let peer_addrs: &[String];
-    let node;
-    if let Some(discover_addr) = matches.value_of("discover_addr") {
-        addr = discover_addr.to_string();
-        node = RemoteNode::new(SocketAddr::from_str(&addr).unwrap(), &[]);
-        addrs = node.read().unwrap().peer_node_addrs().unwrap();
-        peer_addrs = &addrs;
-        info!("Discovered peer_addrs to be {:?}", &peer_addrs);
-    } else {
-        addrs = values_t_or_exit!(matches, "addrs", String);
-        addr = addrs[0].clone();
-        node = RemoteNode::new(SocketAddr::from_str(&addr).unwrap(), &[]);
-        peer_addrs = &addrs[1..];
-    }
+    let addr = value_t_or_exit!(matches, "addr", String);
 
-    let accounts: Vec<_> = if massive_accounts {
-        (0..n).map(|i| format!("near_{}_{}", i, i)).collect()
-    } else {
-        (0..n).map(|i| format!("{}.{}", &prefix, i)).collect()
-    };
-
-    let num_nodes = peer_addrs.len() + 1;
-    let accounts_per_node = accounts.len() / num_nodes;
-    node.write().unwrap().update_accounts(&accounts[0..accounts_per_node]);
-    let mut nodes = vec![node];
-    for (i, addr) in peer_addrs.iter().enumerate() {
-        let node = RemoteNode::new(
-            SocketAddr::from_str(addr).unwrap(),
-            &accounts[((i + 1) * accounts_per_node)..((i + 2) * accounts_per_node)],
-        );
-        nodes.push(node);
-    }
+    let accounts: Vec<_> = (0..n).map(|i| account_k(i as usize)).collect();
+
+    let node = RemoteNode::new(&addr, &accounts);
+    node.write().unwrap().update_accounts(&accounts);
+    let nodes = vec![node];
+
+    use tokio::prelude::*;
+    use tokio::runtime::Runtime;
+    let mut rt = Runtime::new().unwrap();
 
     // Start the executor.
-    let handle = Executor::spawn(nodes, Some(Duration::from_secs(duration)), tps, transaction_type);
-    handle.join().unwrap();
+    let fut = Executor::spawn(nodes, Some(Duration::from_secs(duration)), tps, transaction_type);
+    rt.block_on(fut);
```

Comment:
How can I return a future object in a function and run tokio_runtime.block_on(fut)?